modules/installation-configuration-parameters.adoc

```diff
@@ -178,18 +180,24 @@ endif::[]
 ifeval::["{context}" == "installing-restricted-networks-ibm-power"]
 :ibm-power:
 endif::[]
+ifeval::["{context}" == "installing-azure-stack-hub-default"]
+:ash-default:
+endif::[]
+ifeval::["{context}" == "installing-azure-stack-hub-customizations"]
+:ash-custom:
+endif::[]
 
 [id="installation-configuration-parameters_{context}"]
 = Installation configuration parameters
```

Comment:
There's a section in this section that says:
> **The openshift-install command does not validate field names for parameters. If an incorrect name is specified, the related file or object is not created, and no error is reported. Ensure that the field names for any parameters that are specified are correct.**

That is no longer the case, on all platforms. We do now validate the key names in the install config.

Reply:
Removed the note.